client.go

```diff
@@ -81,30 +81,60 @@ type Client struct {
 	clientConfig
 }
 
-// NewClient returns a new instance to initiate swaps with.
-func NewClient(dbDir string, serverAddress, proxyAddress string, insecure bool,
-	tlsPathServer string, lnd *lndclient.LndServices, maxLSATCost,
-	maxLSATFee btcutil.Amount) (*Client, func(), error) {
+// ClientConfig is the exported configuration structure that is required to
+// instantiate the loop client.
+type ClientConfig struct {
+	// DbDir is the directory where the loopd database resides.
+	DbDir string
```

Comment:
nit: DBDir?

Reply:
I don't know. I don't like those abbreviations to stand out so much. Fixed the TLS one because of the linter, but this one I am getting no linter complaints about. I guess I don't like all those capitals. `RPCServer` vs `RpcServer`. I prefer the latter because it doesn't attract attention needlessly when glancing over the code.

Reply:
It don't think that abbreviated things deserve more attention just because they're abbreviated.

Reply:
Decapitalized.